coalib/output/ConsoleInteraction.py

```diff
@@ -751,3 +753,36 @@ def show_bears(local_bears,
     bears = inverse_dicts(local_bears, global_bears)
 
     print_bears(bears, show_description, show_params, console_printer)
+
+
+def show_language_bears_capabilities(language_bears_capabilities,
+                                     console_printer):
+    """
+    Displays what the languages bears can detect and fix
+
+    :param language_bears_capabilities:
+        Dictionary with languages as keys and their bears capabilities as
+        values. The capabilities are stored in a tuple of two element where the
+        first one represent what the bears can detect, and the second one what
+        they can fix.
+    :param console_printer:
+        Object to print messages on the console.
+    """
+    if not language_bears_capabilities:
+        console_printer.print("No language was passed")
+    else:
+        for language, capabilities in language_bears_capabilities.items():
+            if capabilities[0]:
+                console_printer.print('coala can do the following for ', end='')
+                console_printer.print(language, color="blue")
+                console_printer.print("\tCan detect: ", end='')
```

Comment:
why are you using tabs? We're not using them in other places, rather put spaces.